packages/shared/tailwind/typography.js

```diff
@@ -86,6 +86,11 @@ module.exports = plugin(({ addUtilities }) => {
       'font-size': '0.6875rem',
       'line-height': '1rem',
     },
+
+    '.typo-tera': {
+      'font-size': '5rem',
+      'line-height': '5.375rem',
+    },
```

Comment:
Should actually be in `typography.ts` it should auto generate AFAIK.

Reply:
Seems like we still need this. Nothing happens if I keep it only at the `ts` file.